setup.py

```diff
@@ -1523,7 +1523,8 @@ def detect_sqlite(self):
                 '_sqlite/prepare_protocol.c',
                 '_sqlite/row.c',
                 '_sqlite/statement.c',
-                '_sqlite/util.c', ]
+                '_sqlite/util.c',
+                '_sqlite/blob.c' ]
```

Comment:
Nitpick: Can we add trailing comma here to avoid future noisy diffs?

```py
sqlite_srcs = [
    ...
    '_sqlite/util.c',
    '_sqlite/blob.c'
]
```